src/services/single-pokemon.js

```diff
@@ -0,0 +1,16 @@
+export const getSinglePokemon = async (pokemon) => {
+  const response = await fetch(
+    "https://pokeapi.co/api/v2/pokemon/" + pokemon.toLowerCase()
+  );
+  if (response.status === 404) {
+    throw Error(
+      "Shoot, that Pokemon's not in the Pokedex, please enter a valid Pokemon."
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Ha! Love this!